package.json

```diff
@@ -72,12 +73,11 @@
     "rollup": "^2.41.4",
     "rollup-plugin-typescript2": "^0.30.0",
     "ts-jest": "^26.5.4",
-    "typedoc": "^0.20.32",
+    "typedoc": "^0.22.10",
     "typedoc-plugin-markdown": "^3.3.0",
     "typescript": "^4.2.2"
   },
   "dependencies": {
-    "@rdfjs/types": "^1.0.1",
```

Comment:
You've removed this package from dependencies, but it's present in `src/error.ts`?

Reply:
It's good practice. All types should be dev-dependencies.

Reply:
Aaah, I missed the addition of it in devDependencies.. 🤦‍♀️